adhocracy4/polls/models.py

```diff
@@ -39,7 +39,7 @@ class Poll(module_models.Item):
                                object_id_field='object_pk')
 
     def get_absolute_url(self):
-        return self.module.get_absolute_url()
+        return self.module.get_detail_url
```

Comment:
Yes, it is a property. Maybe shouldn't be, but anyway. https://github.com/liqd/adhocracy4/blob/33e94010d51b5973de3b89bb3e3fb63aa5e9f03f/adhocracy4/modules/models.py#L98